infra/src/main/java/com/redhat/service/smartevents/infra/api/models/responses/BaseResponse.java

```diff
@@ -2,8 +2,10 @@
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
+@Schema(name = "ObjectReference")
```

Comment:
Is it a convention? Should the "root" object be named as `ObjectReference` or are there other reasons?
Btw can you please create a `OpenAPIConstants` class in the `infra` module so that we centralise similar strings?

Reply:
Sadly that is what we got from AppSRE and every project replicates it:

https://github.com/bf2fc6cc711aee1a0c2a/srs-fleet-manager/blob/10f80db6735befba95ded8ba8c61bad824501206/core/src/main/java/org/bf2/srs/fleetmanager/rest/publicapi/beans/ObjectReference.java